test/test_timeline.py

```diff
@@ -44,7 +44,10 @@ def test_timeline(self):
                 hvd.init()
 
                 # Perform a simple allreduce operation
-                hvd.allreduce(torch.tensor([1, 2, 3]), name='test_allreduce')
+                if 'MLSL_ROOT' in os.environ:
```

Comment:
Can we just remove the else statement and always test against float32?  That way we don't need to add additional coupling with MLSL here.

Reply:
Done